<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlaneControlClass</class>
 <widget class="QMainWindow" name="XPlaneControlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlaneControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLat">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>9</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>35</y>
      <width>17</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>61</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt (meters)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAlt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>61</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>87</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heading</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineHeading">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>87</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>113</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePitch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>113</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>139</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRoll">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>139</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSendVEHX">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>165</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send VEHX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSendCMND">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send CMND</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCmdPath">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>200</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDrefValue">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>270</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDrefPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSendDREF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send DREF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDataIndex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDataValue1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSendDATA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send DATA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDataValue2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDataValue3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDataValue4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XPlaneControl.qrc"/>
 </resources>
 <connections/>
</ui>
